### ISSUE TITLE
Solveset: port procedures to solve non linear multivariate equations equations

### ISSUE THREAD

**Issue by hargup** [sympy/sympy#10008]

**Comment by asmeurer** [comment]
@hargup do you have plans to work on this soon? If not, we should remove the 1.0 milestone. 

This is related to https://github.com/sympy/sympy/issues/10006.

**Comment by hargup** [comment]
> @hargup do you have plans to work on this soon?

Nope, I don't have a lot of free time at least till the end of this semester.

> If not, we should remove the 1.0 milestone. 

Done.

**Comment by kshitij10496** [comment]
I would like to attempt this.
